addons/mail/controllers/discuss.py

```diff
@@ -198,6 +198,13 @@ def mail_init_messaging(self, **kwargs):
             return guest.sudo()._init_messaging()
         raise NotFound()
 
+    @http.route('/mail/channel/channel_members', methods=['POST'], type='json', auth='public')
+    def mail_channel_members(self, **kwargs):
+        channel_sudo = request.env['mail.channel'].browse(kwargs['thread_id']).sudo().exists()
+        if channel_sudo:
+            return channel_sudo.load_more_members(kwargs['known_member_ids'])
+        raise NotFound()
```

Comment:
Should check request comes from a member, otherwise this can become a privacy issue